datasetinsights/datasets/simulation/__init__.py

```diff
@@ -0,0 +1,26 @@
+from .captures import Captures
+from .download import (
+    Downloader,
+    _filter_unsuccessful_attempts,
```

Comment:
Since this method is private, it's probably not a good idea to be imported here.

Reply:
its being used other part of the project

Reply:
I didn't see it was used (except for testing) in the current master.

Reply:
I only see this method used in testing.